modules/ossm-installing-istio-ambient-mode.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/install/ossm-istio-ambient-mode.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-istio-ambient-mode_{context}"]
+= Installing Istio ambient mode
+
+You can install {istio} ambient mode on {ocp-product-title} 4.19 or later and {SMProductName} 3.1.0 or later with the required Gateway API custom resource definitions (CRDs).
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.19 or later, which includes the supported {k8s} Gateway API CRDs required for {istio} ambient mode.
+* You have installed {SMProduct} Operator 3.1.0 or later in the {ocp-product-title} cluster.
+* You are logged in to the {ocp-product-title} either through the web console as a user with the `cluster-admin` role, or with the `oc login` command, depending on the installation method.
+
+.Procedure
+
+. Install the {istio} control plane:
+
+.. Create the `istio-system` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc create namespace istio-system
+----
+
+.. Create the `{istio}` resource similar to the following example:
```

Comment:
Is there some way to rewrite this sentence to make it more clear that the file users are creating here is the `istio.yaml` file they're applying in the next step? Assuming that that's what users are doing in these two steps. Maybe something like this:
```suggestion
.. Create an `{istio}` resource named `istio.yaml` similar to the following example:
```

Up to you if you want a different wording, I just think that being explicit here would be more clear for users.